frontend/src/viewer/components/ui/share-modal.ts

```diff
@@ -24,12 +27,21 @@ export class ShareModal extends LitElement {
           <ion-label slot="end"><i class="la la-copy la-2x"></i></ion-label>
         </ion-item>
 
+        <ion-item lines="full">
```

Comment:
We can probably replace this one with ours instead of having the 2 QRs ?

Reply:
I don't understand. Do you want me to remove the QR code with this.link?

Reply:
The main purpose of this QR code is to open the task with XCTrack so the encoded task should be equally good

Reply:
It opens FlyXc, not XCTrack. But I guess it's not that useful. Deleted.